committee-steering/governance/ug-governance.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes User Group (UG)
+
+## Introduction
+The Kubernetes community and ecosystem benefits providing a place for communication and collaboration between users and community members on topics that either do not have deliverables or are outside the scope of the project itself. These topics are not appropriate for forming SIGs, WGs or Committees, which are intended to drive work within the project. User Groups are the venue for this kind of communication and collaboration.
+
+Example User Groups:
+
+- Big Data - Topics related to Big Data integrations.
+- Cloud Providers - Topics related to public/private cloud provider integrations.
+- Machine Learning - Topics related to ML ecosystem integrations.
+- DevOps - Topics related to DevOps tools and deployments.
+
+## UG Goals
+- Facilitate communication between Kubernetes users and contributors on topics pertaining to - the usage of, the extension of and integration with Kubernetes and Kubernetes subprojects.
+
+## UG Anti-Goals
+- Producing or owning Kubernetes project deliverables or assets.
+- Making Kubernetes project decisions.
+- Creating Kubernetes project roles.
```

Comment:
Ownership of any specific topic

Reply:
Claiming ownership of any specific topic